src/client/delphi_epidata.py

```diff
@@ -15,17 +15,10 @@
 from tenacity import retry, stop_after_attempt
 
 from aiohttp import ClientSession, TCPConnector, BasicAuth
-from importlib.metadata import version, PackageNotFoundError
 
 from delphi.epidata.common.logger import get_structured_logger
 
-# Obtain package version for the user-agent. Uses the installed version by
-# preference, even if you've installed it and then use this script independently
-# by accident.
-try:
-    _version = version("delphi-epidata")
-except PackageNotFoundError:
-    _version = "0.script"
+_version = "4.1.20"
```

Comment:
to agree with the change in `__init__.py`:
```suggestion
__version__ = "4.1.20"
```
(and then change the other references to this variable as well)